src/unit_tests/wrappers/libc/stdio_wrappers.c

```diff
@@ -200,7 +219,11 @@ int __wrap__fseeki64(__attribute__ ((__unused__)) FILE *stream, \
      return mock();
 }
 
+extern FILE *__real_popen(const char *command, const char *type);
 FILE *__wrap_popen(const char *command, const char *type) {
+    if(!test_mode){
+        return __real_popen(command, type);
+    }
```

Comment:
We need to add the wrapper for Windows tests. cmocka winagent is not compiling.

Reply:
done

Reply:
According to this check https://github.com/wazuh/wazuh/pull/19746/checks?check_run_id=17919128342, this still is failing

Reply:
![image](https://github.com/wazuh/wazuh/assets/13010397/4b283de5-8a68-49e9-9fb8-f81c04a20612)

Reply:
done